app/models/stripe_wrapper.rb

```diff
@@ -0,0 +1,33 @@
+module StripeWrapper
+  class Charge
+
+    attr_reader :response, :status
+
+    def initialize(response, status)
+      @response = response
+      @status = status
+    end
+
+    def self.create(option={})
+      begin
+        response = Stripe::Charge.create(
+          amount: option[:amount], 
+          currency: 'usd', 
+          source: option[:source],
+          description: option[:description]
+        )
```

Comment:
is this some kind of code smell? because I feel it is more easy to read in this style than put it in one line.